tic-tac-toe/src/components/control-pannel/control-pannel.js

```diff
@@ -0,0 +1,68 @@
+import React, {Component} from 'react';
+import './control-pannel.css';
+
+const GameModePannel = ({activeGameMode, onChangeMode, outerClassName}) => {
+    const innerClassName = 'mode-panel';
```

Comment:
don't create a variable for constant value inside render, it makes sense to keep it in global context at all or just use it like @Dev1LtJ proposed

Reply:
I fixed it.
```suggestion
const GameModePannel = ({activeGameMode, onChangeMode}) => {
  const buttons = buttonsInfo.map(({name, label}) => {
    const isActive = (name === activeGameMode);
    let btnClassName = classes.Button;
    btnClassName += (isActive) ? ` ${classes.ActiveButton}` : '';
```